contract/src/main.rs

```diff
@@ -1560,57 +1561,60 @@ pub extern "C" fn set_token_metadata() {
 
 #[no_mangle]
 pub extern "C" fn migrate() {
-    let requires_rlo_migration = utils::requires_rlo_migration();
+    let reporting_mode = match runtime::get_key(REPORTING_MODE).is_some() {
```

Comment:
This logic seems incorrect, with this, we automatically treat ALL 1_0_0 contracts as opting into NoLookup which is incorrect

Reply:
Ok thanks can you please check that commit ? https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/218/commits/ebf1f2dc170f80c28414a084ba191a5fe07809fd